static/src/javascripts/bootstraps/standard/main.js

```diff
@@ -177,6 +177,14 @@ const bootStandard = (): void => {
     // Set adtest query if url param declares it
     setAdTestCookie();
 
+    // Remove the old ad-free cookie - we can delete this line in a few days
+    removeCookie('GU_AFU');
```

Comment:
what happens if return after this has been removed and still have the old cookie?